### ISSUE TITLE
Specify explicit version of the base image

### ISSUE THREAD

**Issue by osvso** [tracer0tong/android-emulator#16]
Currently the Dockerfile is based on latest ubuntu image
`FROM ubuntu`
and therefore may become unstable at some point in time. 
Please consider changing to specific verified version of `ubuntu` ex. `ubuntu:16.04`.

**Comment by tracer0tong** [comment]
Accordingly documentation on Dockerhub for `ubuntu:latest` container:

> The ubuntu:latest tag points to the "latest LTS", since that's the version recommended for general use.

I'm not sure that it's so unsafe, because usually I'm trying to test container before pushing to Dockerhub.
Do you have some other concerns?

**Comment by osvso** [comment]
Yes, my only concern is that the underlaying ubuntu changes in between versions and may require some additional repositories or libraries in order to get the emulator working. If someone will clone you repo and build the image locally he may find it not working, which shouldn't be the case if the image is pulled from docker repo and tested, as you mentioned, up-front.

**Comment by tracer0tong** [comment]
@nightscape what do you think about it?
